diff --git a/BoardConfig.mk b/BoardConfig.mk
index a9590e1..4bb6279 100644
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -260,7 +260,7 @@ WIFI_DRIVER_OPERSTATE_PATH := "/sys/class/net/wlan0/operstate"
 WIFI_HIDL_FEATURE_DUAL_INTERFACE := true
 
 # Pre-opt SystemUI
-PRODUCT_DEXPREOPT_SPEED_APPS += SystemUI
+# PRODUCT_DEXPREOPT_SPEED_APPS += SystemUI
 
 # inherit from the proprietary version
 -include vendor/asus/X00TD/BoardConfigVendor.mk
